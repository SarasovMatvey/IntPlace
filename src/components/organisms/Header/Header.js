import { NavLink } from "react-router-dom";
import { Menu } from "semantic-ui-react";
import "./Header.sass";
import logo from "./img/logo.png";

function Header() {
  return (
    <Menu fixed="top" className="header">
      <Menu.Item>
        <img className="header__logo" src={logo} />
      </Menu.Item>
      <Menu.Item>
        <NavLink
          className="header__link"
          activeClassName="header__link_active"
          exact
          to="/"
        >
          Main
        </NavLink>
      </Menu.Item>
    </Menu>
  );
}

export default Header;
