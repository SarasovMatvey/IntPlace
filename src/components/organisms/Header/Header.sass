.header
  &__logo
    width: 130px !important
    height: 50px
    object-fit: contain
  &__link
    color: #000
  &__link_active
    background: linear-gradient(to bottom, #ff9800 0%, #ff9800 100%)
    background-position: 0 100%
    background-repeat: repeat-x
    background-size: 3px 3px
    text-decoration: none
  