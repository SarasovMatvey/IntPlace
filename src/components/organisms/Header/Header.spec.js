import Header from "./Header";

const setUp = props => shallow(<Header {...props} />);

describe("Header", () => {
  let component;

  beforeEach(() => {
    component = setUp();
  });

  it("should renders properly", () => {
    expect(component).toMatchSnapshot();
  });
});
