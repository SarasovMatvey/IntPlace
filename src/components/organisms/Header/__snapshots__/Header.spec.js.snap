// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should renders properly 1`] = `
<Menu
  className="header"
  fixed="top"
>
  <MenuItem>
    <img
      alt="logo"
      className="header__logo"
      src="logo.png"
    />
  </MenuItem>
  <MenuItem>
    <NavLink
      activeClassName="header__link_active"
      className="header__link"
      exact={true}
      to="/"
    >
      Main
    </NavLink>
  </MenuItem>
  <MenuItem>
    <NavLink
      activeClassName="header__link_active"
      className="header__link"
      to="/favorites"
    >
      Favorites
    </NavLink>
  </MenuItem>
</Menu>
`;
